src/bosh_azure_cpi/lib/cloud/azure/storage/blob_manager.rb

```diff
@@ -239,7 +239,11 @@ def list_blobs(storage_account_name, container_name, prefix = nil)
         loop do
           options = merge_storage_common_options(options)
           @logger.info("list_blobs: Calling list_blobs(#{container_name}, #{options})")
-          temp = @blob_service_client.list_blobs(container_name, options)
+          begin
+            temp = @blob_service_client.list_blobs(container_name, options)
+          rescue StandardError => e
+            return [] if e.inspect.include?('ContainerNotFound')
```

Comment:
better to log the e.inspect. first and check whether it's nil.

Reply:
This has been removed from this PR.